buildkite/src/Lib/Cmds.dhall

```diff
@@ -43,8 +43,9 @@ let module = \(environment : List Text) ->
         (docker.extraEnv # environment)
     let outerDir : Text =
       "/var/buildkite/builds/\\\$BUILDKITE_AGENT_NAME/\\\$BUILDKITE_ORGANIZATION_SLUG/\\\$BUILDKITE_PIPELINE_SLUG"
+    let sharedDir : Text = "/var/buildkite/shared"
     in
-    { line = "docker run -it --rm --init --volume ${outerDir}:/workdir --workdir /workdir${envVars} ${docker.image} /bin/sh -c '${inner.line}'"
+    { line = "docker run -it --rm --init --volume /var/buildkite:${sharedDir} --volume ${outerDir}:/workdir --workdir /workdir${envVars} ${docker.image} /bin/sh -c '${inner.line}'"
```

Comment:
TODO: replace w/ `/var/buildkite/shared` once https://github.com/buildkite/charts/pull/71 is merged